### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by GreyPK** [hop-protocol/hop-airdrop#346]
# Related Addresses

24 related addresses

```
0x858c5a5ddda4104d6efd731cfef790edcff87841
0xa1a52925369c3870aacf26decf7a8727e5a47fb7
0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74
0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39
0xca625beda8d0aade9a89c4050353d81ed6dc1f7a
0xc9c0e29f089ae04623536f3c9d32b557a323656e
0x76967322893cef3e8efa713daf1dc8347e081083
0x20b2e86d545768305d9629176fad5c0764fabb42
0x74a4085839f18c19d6ce6af367b4bc458aec350c
0x45e81cf357f3f66a295e8a85e60832810801c660
0x1815eb802bc359e5a58cb9a1c19612f5620a2629
0xbd48b5051bb74030d23676058ef91e19b81e3f9f
0x2f7dbf19c6901ecbf5b36f2c6667883fc533f8c6
0xe636c37c8161d1097090534e294cabae78dcdca3
0x8b647f694db9c159c27a0c044550b81082f5df18
0xf7ce5f7a13dee8ba402fa750088e478db1f2b4f4
0xc647500d4778e9b41fa4423c4264b7185952e741
0x95f4b2aa12f3a85e2f3922352f0cb8e7b4b96cbc
0x26c35cf6a0ed1a821b25767548e16f324a3e2622
0x4bbd5fce20f25d6f185ff15ad037eb999fecbcb4
0x3a91e8159fa5d0266475629a4704ee28e6396667
0xfa18743f907ffe9bff1a021b617e5134a4693390
0x813a9e2c84d542392e24ad371a0f37e9164caeb6
0x393ba4eb65519492e9c94a7adf0097006b70603f
```

# Reasoning

1) These addresses often make transactions between themselves eth on mainnet, stablecoins on polygon and gnosis.
2) All wallets used HOP 14-19 Nov 2021 to bridge stablecoins to Polygon/Gnosis or from Polygon/Gnosis.
3) Each address used HOP swapAndSend 2-16 times in this day. Total volume $1200-$16000.

I generated a graph of eth transfers on Mainnet. And found 22 addresses used that pattern.
![main_potencial_3_20pluspng](https://user-images.githubusercontent.com/9148071/168495455-fb3b4e70-188e-4bd4-9e4f-49a264bbd727.png)

0xa1a52925369c3870aacf26decf7a8727e5a47fb7 14-16 Nov 2021, 5 txs, total ~$1200
0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74 14-16 Nov 2021, 5 txs, total ~$1670
0x76967322893cef3e8efa713daf1dc8347e081083 14-15 Nov 2021, 5 txs, total ~$7325
0xe636c37c8161d1097090534e294cabae78dcdca3 15 Nov 2021, 7 txs, total ~$10220
0x858c5a5ddda4104d6efd731cfef790edcff87841 15 Nov 2021, 7 txs, total ~$9800
0x45e81cf357f3f66a295e8a85e60832810801c660 15 Nov 2021, 7 txs, total ~$9940
0x74a4085839f18c19d6ce6af367b4bc458aec350c 16 Nov 2021, 8 txs, total ~$11880
0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39 16-17 Nov 2021, 6 txs, total ~$5850
0xfa18743f907ffe9bff1a021b617e5134a4693390 16-17 Nov 2021, 9 txs, total ~$11610
0x95f4b2aa12f3a85e2f3922352f0cb8e7b4b96cbc 17 Nov 2021, 9 txs, total ~$10400
0x26c35cf6a0ed1a821b25767548e16f324a3e2622 17 Nov 2021, 11 txs, total ~$12000
0x4bbd5fce20f25d6f185ff15ad037eb999fecbcb4 17 Nov 2021, 13 txs, total ~$16000
0x20b2e86d545768305d9629176fad5c0764fabb42 17 Nov 2021, 8 txs, total ~$9000
0xf7ce5f7a13dee8ba402fa750088e478db1f2b4f4 18 Nov 2021, 16 txs, total ~$17300
0xc647500d4778e9b41fa4423c4264b7185952e741 18 Nov 2021, 6 txs, total ~$6300
0x1815eb802bc359e5a58cb9a1c19612f5620a2629 18 Nov 2021, 4 txs, total ~$4000
0x8b647f694db9c159c27a0c044550b81082f5df18 18-19 Nov 2021, 6 txs, total ~$1158
0xbd48b5051bb74030d23676058ef91e19b81e3f9f 18-19 Nov 2021, 12 txs, total ~$11600
0xca625beda8d0aade9a89c4050353d81ed6dc1f7a 19 Nov 2021, 6 txs, total ~$5640
0xc9c0e29f089ae04623536f3c9d32b557a323656e 19 Nov 2021, 5 txs, total ~$4625
0x2f7dbf19c6901ecbf5b36f2c6667883fc533f8c6 19 Nov 2021, 2 txs, total ~$3800
0x3a91e8159fa5d0266475629a4704ee28e6396667 19 Nov 2021, 8 txs, total ~$14900
0x813a9e2c84d542392e24ad371a0f37e9164caeb6 16 Nov 2021, 6 txs, total ~$6000
0x393ba4eb65519492e9c94a7adf0097006b70603f 17 Nov 2021, 6 txs, total ~$5760

https://explorer.hop.exchange/?account=0x858c5a5ddda4104d6efd731cfef790edcff87841
https://explorer.hop.exchange/?account=0xa1a52925369c3870aacf26decf7a8727e5a47fb7
https://explorer.hop.exchange/?account=0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74
https://explorer.hop.exchange/?account=0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39
https://explorer.hop.exchange/?account=0xca625beda8d0aade9a89c4050353d81ed6dc1f7a
https://explorer.hop.exchange/?account=0xc9c0e29f089ae04623536f3c9d32b557a323656e
https://explorer.hop.exchange/?account=0x76967322893cef3e8efa713daf1dc8347e081083
https://explorer.hop.exchange/?account=0x20b2e86d545768305d9629176fad5c0764fabb42
https://explorer.hop.exchange/?account=0x74a4085839f18c19d6ce6af367b4bc458aec350c
https://explorer.hop.exchange/?account=0x45e81cf357f3f66a295e8a85e60832810801c660
https://explorer.hop.exchange/?account=0x1815eb802bc359e5a58cb9a1c19612f5620a2629
https://explorer.hop.exchange/?account=0xbd48b5051bb74030d23676058ef91e19b81e3f9f
https://explorer.hop.exchange/?account=0x2f7dbf19c6901ecbf5b36f2c6667883fc533f8c6
https://explorer.hop.exchange/?account=0xe636c37c8161d1097090534e294cabae78dcdca3
https://explorer.hop.exchange/?account=0x8b647f694db9c159c27a0c044550b81082f5df18
https://explorer.hop.exchange/?account=0xf7ce5f7a13dee8ba402fa750088e478db1f2b4f4
https://explorer.hop.exchange/?account=0xc647500d4778e9b41fa4423c4264b7185952e741
https://explorer.hop.exchange/?account=0x95f4b2aa12f3a85e2f3922352f0cb8e7b4b96cbc
https://explorer.hop.exchange/?account=0x26c35cf6a0ed1a821b25767548e16f324a3e2622
https://explorer.hop.exchange/?account=0x4bbd5fce20f25d6f185ff15ad037eb999fecbcb4
https://explorer.hop.exchange/?account=0x3a91e8159fa5d0266475629a4704ee28e6396667
https://explorer.hop.exchange/?account=0xfa18743f907ffe9bff1a021b617e5134a4693390
https://explorer.hop.exchange/?account=0x813a9e2c84d542392e24ad371a0f37e9164caeb6
https://explorer.hop.exchange/?account=0x393ba4eb65519492e9c94a7adf0097006b70603f
# Methodology

I checked eth transactions with etherscan API and filtered transactions, where 10 recipients addresses are eligible for the airdrop, that make transactions between themselves. Then I checked when and how these wallets interact with HOP. Node js code available on request.

# Rewards Address

0x7d3f6048091aEdE6198736b8C6ac527c314e25ff

**Comment by shanefontaine** [comment]
@GreyPK Thank you for the submission! Most of these addresses look great!

Would you be able to provide slightly more proof around the 5 addresses in the image that are not connected in the inner web? They don't seem to follow the same _exact_ patterns that the others do (although the connection looks legitimate).

The addresses are:

```
0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39
0xca625beda8d0aade9a89c4050353d81ed6dc1f7a
0xc9c0e29f089ae04623536f3c9d32b557a323656e
0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74
0xa1a52925369c3870aacf26decf7a8727e5a47fb7
```

**Comment by GreyPK** [comment]
This is a graph of matic transfers:
![346](https://user-images.githubusercontent.com/9148071/168721058-02b66ddf-4252-47ae-acf3-dc77a3a31505.png)
1) 0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74 received USDC from 0xa1a52925369c3870aacf26decf7a8727e5a47fb7
And send USDT to 0x813a9e2c84d542392e24ad371a0f37e9164caeb6
https://polygonscan.com/tx/0xe8051bdbed5ecbcfc3c68db02df53c40d3305d0255e6bd53a7aab58436e4c1ac
https://polygonscan.com/tx/0xf72d8b1c1cfbbc8ddbdb1ad96fef344918761d04999a8688d3d759277241608a

2) 0x813a9e2c84d542392e24ad371a0f37e9164caeb6 send USDC to 0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39
and received Matic from
0xa1a52925369c3870aacf26decf7a8727e5a47fb7,
0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74,
0x74a4085839f18c19d6ce6af367b4bc458aec350c
https://polygonscan.com/tx/0xd6fe7b57c9fc3d31e1651d8e7114a1ec56ab73f0d345035a5bfbb99c0a0603c7
https://polygonscan.com/tx/0x8697853cabc6dc7a8288f52ff91cb565731fd8c453aacda8189c9f214cdd1872
https://polygonscan.com/tx/0x0216f104a0538ee5c7ddfec7d1b96cfa0bc85070bedf252f07aeaec037be657f
https://polygonscan.com/tx/0xd920bd8d09403519b4d6da1d2884402523f2e852620c4820aa86ec6a7fae6953

3) 0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39 send USDC to 0x393ba4eb65519492e9c94a7adf0097006b70603f
https://polygonscan.com/tx/0x858b8cd16e3119136421c6978666988857bc8e12c09e42bc50cda436e636b60f

4) 0xca625beda8d0aade9a89c4050353d81ed6dc1f7a send USDT to 0xc9c0e29f089ae04623536f3c9d32b557a323656e
https://polygonscan.com/tx/0x385f7c9f66aefaa3c3532fe4edc67d0d5577409ec8401624698212c1de9d867f

So on mainnet 0x858c5a5ddda4104d6efd731cfef790edcff87841 send eth to 0xa1a52925369c3870aacf26decf7a8727e5a47fb7, wich connected to 0x813a9e2c84d542392e24ad371a0f37e9164caeb6, which connected to 0x74a4085839f18c19d6ce6af367b4bc458aec350c.
https://etherscan.io/tx/0xe9317ad1627dfafba853f2862ef96107dd13be629570ad6cff428e4c1dfce8cb

In other words, the large group is connected to the small group on mainnet through 0x858c5a5ddda4104d6efd731cfef790edcff87841 and on Polygon throw 0x74a4085839f18c19d6ce6af367b4bc458aec350c

Also found 2 more related addresses
```
0x813a9e2c84d542392e24ad371a0f37e9164caeb6
0x393ba4eb65519492e9c94a7adf0097006b70603f
```
0x813a9e2c84d542392e24ad371a0f37e9164caeb6 16 Nov 2021, 6 txs, total ~$6000
0x393ba4eb65519492e9c94a7adf0097006b70603f 17 Nov 2021, 6 txs, total ~$5760

**Comment by zlgitol** [comment]
https://polygonscan.com/tx/0x8697853cabc6dc7a8288f52ff91cb565731fd8c453aacda8189c9f214cdd1872
https://polygonscan.com/tx/0xd920bd8d09403519b4d6da1d2884402523f2e852620c4820aa86ec6a7fae6953
are obviously not correct. 
You should not connect irrelevant addresses by any transactions.

**Comment by GreyPK** [comment]
[zlgitol](https://github.com/zlgitol), 0x813a9e2c84d542392e24ad371a0f37e9164caeb6 received Matic from
0xa1a52925369c3870aacf26decf7a8727e5a47fb7 (small group),
0x74a4085839f18c19d6ce6af367b4bc458aec350c (large group)
In almost same time and same amount, for pay transactions fees because he swaped all available matic to USDC.
https://polygonscan.com/txs?a=0x813a9e2c84d542392e24ad371a0f37e9164caeb6&f=3
![proof](https://user-images.githubusercontent.com/9148071/168736707-a0023467-a17e-4ebc-bb31-3799302211df.png)
https://debank.com/profile/0x813a9e2c84d542392e24ad371a0f37e9164caeb6/history?chain=matic&token=matic_MATIC
![proof2](https://user-images.githubusercontent.com/9148071/168737782-3acc5c34-e42e-4125-85af-7ff005754c12.png)

All transactions I found from between **large** group and **small** group:
0x858c5a5ddda4104d6efd731cfef790edcff87841 send cUSD and ETH to 0xa1a52925369c3870aacf26decf7a8727e5a47fb7
https://etherscan.io/tx/0xe9317ad1627dfafba853f2862ef96107dd13be629570ad6cff428e4c1dfce8cb
https://explorer.celo.org/tx/0x0b01e671a155c05b2bfe8c581350eafeb61cf4acd4d74e10596e3247ec4aa193/token-transfers
https://explorer.celo.org/tx/0x72a099b9d59e7150dc65c11f6e77cea034537505bb0b1532a620db25f799a5cd/token-transfers

0x20b2e86d545768305d9629176fad5c0764fabb42 received and send CELO from 0xa1a52925369c3870aacf26decf7a8727e5a47fb7
https://explorer.celo.org/tx/0x56b54ea7c3fcdcd3ef3467bc304bab1857f6385bea6e7b8f73cd239cc3bf7cff/token-transfers
https://explorer.celo.org/tx/0xb2308ab68fffd201a856a9d9c63a43a976ce9fd695bf7d397489da544b6fe1e9/token-transfers

0xe636c37c8161d1097090534e294cabae78dcdca3 received and send tokens from 0xa1a52925369c3870aacf26decf7a8727e5a47fb7
https://polygonscan.com/tx/0xe8216bf6dd6b71d50ecc076fa64756232fb9534143da112afca3643f91ec027f
https://polygonscan.com/tx/0x9adbf117e06a4d644a01f390ae62b2089be50ecc9d0819412bdabbdcf69e528c
https://polygonscan.com/tx/0xb0e3bf6c6c37400cd5878965f854ff8be84e3bafd0826571bdf8cbb405699151
https://explorer.celo.org/tx/0x739ce113a648d4160d82d1e5be1423824c597a5e178eae9f75f950a82138a05b
https://hecoinfo.com/tx/0x77285a6ba5f27a59157294688f17555592b79a6632c0172d03dec39dcf39c071
https://blockscout.com/poa/xdai/tx/0xa3dfdc988609975eca5eb2af4afde12cfd8a04bebeabc559faa93f15dae0d878
https://arbiscan.io/tx/0xbc9fbc9dc70b751fdd476d10f5d00d1f13b7621d4001cb9ce1de7d7b394bda5b
https://hecoinfo.com/tx/0x95957c2e823485699386db9b6915e41128f20279cc07ad94dc0e0446df1018ce
https://bscscan.com/tx/0xe4336df9d09074011e0ddbe68d8c3d4ed80020683f0a4b0eedb906d49edb9c3d
https://snowtrace.io/tx/0xc6cb00bf09d460640d58487dd57c2d98a5b2b242322cd208a591324440fa7bc4

So the large group is connected to the small group through:
0x858c5a5ddda4104d6efd731cfef790edcff87841,
0x74a4085839f18c19d6ce6af367b4bc458aec350c, 
0x20b2e86d545768305d9629176fad5c0764fabb42,
0xe636c37c8161d1097090534e294cabae78dcdca3

**Comment by GreyPK** [comment]
I counted the number of HOP transactions in each hour and made a chart, GMT+6.
![chart](https://user-images.githubusercontent.com/9148071/168778942-a0dc9801-e6d5-423b-9aaf-cc41d8191db9.png)

There were a total of 180 transactions: 41 of small group and 139 from large group. You can see that the groups correlate with each other and from 23:30 to 7:40 there was no transaction(earliest transaction was at 2021-11-15T07:42, GMT+6 by 0x76967322893cef3e8efa713daf1dc8347e081083, latest was at 2021-11-17T23:26, GMT+6 by 0x20b2e86d545768305d9629176fad5c0764fabb42). It can be assumed that the user is asleep at this time.
[issue346_timings.xlsx](https://github.com/hop-protocol/hop-airdrop/files/8707145/issue346_timings.xlsx) 
[346_timings.txt](https://github.com/hop-protocol/hop-airdrop/files/8707171/346_timings.txt)

**Comment by GreyPK** [comment]
Transaction of small group about token transfers in circle:
0xa1a52925369c3870aacf26decf7a8727e5a47fb7 send 281 USDC + 141 Matic to 0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74
https://polygonscan.com/tx/0x6b0df7daf215c10d55631e3bc35c7a359351c09c3e4c93aaa7821c13bc734229
https://polygonscan.com/tx/0xe8051bdbed5ecbcfc3c68db02df53c40d3305d0255e6bd53a7aab58436e4c1ac

0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74 send 1000 USDT to 0x813a9e2c84d542392e24ad371a0f37e9164caeb6
https://polygonscan.com/tx/0xf72d8b1c1cfbbc8ddbdb1ad96fef344918761d04999a8688d3d759277241608a

0x813a9e2c84d542392e24ad371a0f37e9164caeb6 send 985 USDC to 0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39
https://polygonscan.com/tx/0xd6fe7b57c9fc3d31e1651d8e7114a1ec56ab73f0d345035a5bfbb99c0a0603c7

0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39 send 968 USDC to 0x393ba4eb65519492e9c94a7adf0097006b70603f
https://polygonscan.com/tx/0x858b8cd16e3119136421c6978666988857bc8e12c09e42bc50cda436e636b60f

0x393ba4eb65519492e9c94a7adf0097006b70603f swap USDC to matic and send 598 matic to 0xca625beda8d0aade9a89c4050353d81ed6dc1f7a
https://polygonscan.com/tx/0x218daaefbc734241e6c860d36f6acb1097c087f61adc658cc53629719170d124

0xca625beda8d0aade9a89c4050353d81ed6dc1f7a send 930 USDT to 0xc9c0e29f089ae04623536f3c9d32b557a323656e
https://polygonscan.com/tx/0x385f7c9f66aefaa3c3532fe4edc67d0d5577409ec8401624698212c1de9d867f

0xc9c0e29f089ae04623536f3c9d32b557a323656e swap USDT to matic and send 566 matic to 0x23a3d91e7f6d2c443c1ff0b5333325b615b3d7c3 => 0x98c6ed552d0315622f9b2ba735e9058b21d7089f => 0x8fc5f1fd3932c98e4b30b48424115155fa901bef => 0x970ae3b7ad1c921d59f4a67f86e705a45c972fe9 => 0xa1a52925369c3870aacf26decf7a8727e5a47fb7
https://polygonscan.com/tx/0x43e0d131b7c33314db221d347be5eefa14c1f20d6e43375089ef81a600aabe20
https://polygonscan.com/tx/0x73b006d354185d5e5f355056af6a3ce3bc1882f09ca297dc20057417487652e2
https://polygonscan.com/tx/0xb9d678fe9ff6007ff05b76e966822d9389dfddd42269cd8eb3eccae2068e086c
https://polygonscan.com/tx/0xb787fb66fb52c9a22c68b7def592c1e3a7908d860824e762f864c7e5440c9d35
https://polygonscan.com/tx/0x756be988b916e1d35f78ceec2bcce03326d388046d4dd95ee51ad12ed6332872

**Comment by shanefontaine** [comment]
Thank you for your report @GreyPK. We have verified that the addresses in this report are Sybil attackers.

The report included 24 eligible addresses as Sybil attackers which means you are eligible for 7951.132882408009164021 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).

The qualified addresses are as follows:

```
0x858c5a5ddda4104d6efd731cfef790edcff87841
0xa1a52925369c3870aacf26decf7a8727e5a47fb7
0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74
0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39
0xca625beda8d0aade9a89c4050353d81ed6dc1f7a
0xc9c0e29f089ae04623536f3c9d32b557a323656e
0x76967322893cef3e8efa713daf1dc8347e081083
0x20b2e86d545768305d9629176fad5c0764fabb42
0x74a4085839f18c19d6ce6af367b4bc458aec350c
0x45e81cf357f3f66a295e8a85e60832810801c660
0x1815eb802bc359e5a58cb9a1c19612f5620a2629
0xbd48b5051bb74030d23676058ef91e19b81e3f9f
0x2f7dbf19c6901ecbf5b36f2c6667883fc533f8c6
0xe636c37c8161d1097090534e294cabae78dcdca3
0x8b647f694db9c159c27a0c044550b81082f5df18
0xf7ce5f7a13dee8ba402fa750088e478db1f2b4f4
0xc647500d4778e9b41fa4423c4264b7185952e741
0x95f4b2aa12f3a85e2f3922352f0cb8e7b4b96cbc
0x26c35cf6a0ed1a821b25767548e16f324a3e2622
0x4bbd5fce20f25d6f185ff15ad037eb999fecbcb4
0x3a91e8159fa5d0266475629a4704ee28e6396667
0xfa18743f907ffe9bff1a021b617e5134a4693390
0x813a9e2c84d542392e24ad371a0f37e9164caeb6
0x393ba4eb65519492e9c94a7adf0097006b70603f
```

**Comment by shanefontaine** [comment]
In addition to the data provided, we also see some more compelling evidence:

* Almost all addresses exist in a chain on FTM
* They all share very similar behaviors on Hop protocol

address | ensName | total | mainnet | arbitrum | optimism | polygon | xdai | totalVolume
-- | -- | -- | -- | -- | -- | -- | -- | --
0xf7ce5f7a13dee8ba402fa750088e478db1f2b4f4 |   | 16 | 0 | 0 | 0 | 8 | 8 | $17,577.71
0x4bbd5fce20f25d6f185ff15ad037eb999fecbcb4 |   | 13 | 0 | 0 | 0 | 7 | 6 | $16,251.75
0x3a91e8159fa5d0266475629a4704ee28e6396667 |   | 8 | 0 | 0 | 0 | 4 | 4 | $15,157.16
0x26c35cf6a0ed1a821b25767548e16f324a3e2622 |   | 11 | 0 | 0 | 0 | 6 | 5 | $12,098.97
0x74a4085839f18c19d6ce6af367b4bc458aec350c |   | 9 | 0 | 0 | 0 | 5 | 4 | $12,029.55
0xbd48b5051bb74030d23676058ef91e19b81e3f9f |   | 12 | 0 | 0 | 0 | 6 | 6 | $11,711.10
0xfa18743f907ffe9bff1a021b617e5134a4693390 |   | 9 | 0 | 0 | 0 | 4 | 5 | $11,696.90
0x8b647f694db9c159c27a0c044550b81082f5df18 |   | 6 | 0 | 0 | 0 | 3 | 3 | $11,693.43
0xe636c37c8161d1097090534e294cabae78dcdca3 |   | 8 | 0 | 0 | 0 | 3 | 5 | $10,830.81
0x95f4b2aa12f3a85e2f3922352f0cb8e7b4b96cbc |   | 9 | 0 | 0 | 0 | 4 | 5 | $10,586.86
0x45e81cf357f3f66a295e8a85e60832810801c660 | *honda-motor.eth | 7 | 0 | 0 | 0 | 4 | 3 | $10,135.59
0x858c5a5ddda4104d6efd731cfef790edcff87841 | jiaguwen.eth | 7 | 0 | 0 | 0 | 3 | 4 | $9,906.62
0x20b2e86d545768305d9629176fad5c0764fabb42 |   | 8 | 0 | 0 | 0 | 4 | 4 | $9,152.24
0x76967322893cef3e8efa713daf1dc8347e081083 |   | 5 | 0 | 0 | 0 | 3 | 2 | $7,446.94
0xc647500d4778e9b41fa4423c4264b7185952e741 |   | 6 | 0 | 0 | 0 | 3 | 3 | $6,386.00
0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39 |   | 6 | 0 | 0 | 0 | 3 | 3 | $5,883.47
0xca625beda8d0aade9a89c4050353d81ed6dc1f7a |   | 6 | 0 | 0 | 0 | 3 | 3 | $5,679.36
0xc9c0e29f089ae04623536f3c9d32b557a323656e |   | 5 | 0 | 0 | 0 | 3 | 2 | $4,646.79
0x1815eb802bc359e5a58cb9a1c19612f5620a2629 |   | 4 | 0 | 0 | 0 | 2 | 2 | $4,046.52
0x2f7dbf19c6901ecbf5b36f2c6667883fc533f8c6 |   | 2 | 0 | 0 | 0 | 1 | 1 | $3,855.11
0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74 |   | 7 | 0 | 1 | 0 | 3 | 3 | $1,682.97
0xa1a52925369c3870aacf26decf7a8727e5a47fb7 |   | 5 | 0 | 0 | 0 | 2 | 3 | $1,206.11

**Comment by jukjh** [comment]
Why？

**Comment by jukjh** [comment]
I am a real user, why attack my address? You are reporting maliciously.

**Comment by jukjh** [comment]
Is this how you treat your users? Just concatenate a few addresses, and you will be spoofed by you? I like cross-chain Matic and Xdai。What does it have to do with you? Aren't your products just for us to use?

**Comment by jukjh** [comment]
You are maliciously linking accounts. And then benefit those whistleblowers.

**Comment by zlgitol** [comment]
> You are maliciously linking accounts. And then benefit those whistleblowers.

I also think that transactions whose values are 0.1 or 0.01 MATIC cannot be considered in account linking.

**Comment by jukjh** [comment]
You don't check hundreds or thousands of addresses.Check out a dozen or so addresses，I just want to ask you. Do you have more than a dozen bank cards or credit cards?

**Comment by jukjh** [comment]
You don't check hundreds or thousands of addresses.Check out a dozen or so addresses，I just want to ask you. Do you have more than a dozen bank cards or credit cards?

**Comment by jukjh** [comment]
You don't check hundreds or thousands of addresses.Check out a dozen or so addresses，I just want to ask you. Do you have more than a dozen bank cards or credit cards?

**Comment by jukjh** [comment]
0xa1a52925369c3870aacf26decf7a8727e5a47fb7
0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74
0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39
0xca625beda8d0aade9a89c4050353d81ed6dc1f7a
0xc9c0e29f089ae04623536f3c9d32b557a323656e
0x813a9e2c84d542392e24ad371a0f37e9164caeb6
0x393ba4eb65519492e9c94a7adf0097006b70603f
these are not my addresses

**Comment by mattick27** [comment]
@shanefontaine I mention in group_03 #241

**Comment by shanefontaine** [comment]
@mattick27 group_03 of #241 only had 17 eligible addresses at the time of submission. All submissions of 20+ addresses were considered prior to the announcement of [the 10+ submission acceptance](https://discord.com/channels/789310208413270078/802765167552626719/976587814831546448).

**Comment by wangchen54** [comment]
Hello, I don't know what technical means you used to get my address, but this is not exactly my address, I only have 7 addresses, as for the association, I don't know how to explain it, you can check the detailed information, only Seven addresses are mine, I don't know the others, and I don't know how they are related. Please give me justice. I have no malicious intentions to get the airdrop. Below are my seven addresses.
0xA1a52925369C3870aACF26dEcf7A8727e5a47FB7

0x0aCA91069e5Ccf2d16859E9f5E00AF51fe17fD74

0x813A9e2c84d542392E24Ad371A0f37e9164CaEB6

0x5e2e6606d3A32F7A87e39F47a1F4AFFC01B11E39

0x393BA4eb65519492E9C94A7aDf0097006b70603F

0xcA625BEdA8D0AaDE9A89C4050353d81ED6dc1F7A

0xC9C0e29f089Ae04623536F3C9d32b557A323656E

**Comment by wangchen54** [comment]
According to your logic, no matter how long ago there was a hash association, it is defined as the same account holder, which is not reasonable. For example, A knows B, B knows C, C knows D, and so on. People in the whole world are the same person, I have made my address clear, I hope you will verify it carefully, I only have seven addresses, I don’t know the others, you can query the hash, where are we related, and the repetition of the relationship Do, give me justice, this is the result of my three days, and I have paid hundreds of dollars in fees, give me justice, I am not a malicious airdropper.

**Comment by wangchen54** [comment]
>这7个地址是我的，你是谁啊？咱们认识吗？为啥会混到一块?

**Comment by wangchen54** [comment]
> 0xa1a52925369c3870aacf26decf7a8727e5a47fb7 0x0aca91069e5ccf2d16859e9f5e00af51fe17fd74 0x5e2e6606d3a32f7a87e39f47a1f4affc01b11e39 0xca625beda8d0aade9a89c4050353d81ed6dc1f7a 0xc9c0e29f089ae04623536f3c9d32b557a323656e 0x813a9e2c84d542392e24ad371a0f37e9164caeb6 0x393ba4eb65519492e9c94a7adf0097006b70603f these are not my addresses
这7个地址是我的，你是谁啊？咱们认识吗？为啥会混到一块?

**Comment by wangchen54** [comment]
It doesn't matter if I ask the official to give justice, and it doesn't matter if I don't give rewards. Please compensate me for my losses. I am not a malicious attacker, and I have not violated any official rules.
